tipocket-ctl/tpctl/params.py

```diff
@@ -8,7 +8,7 @@ def parse_params(params):
     # convert parameters to the format that tipocket test case recognize
     case_params = {}
     for key, value in params.items():
-        if key in ['build_image', 'subscriber', 'feature']:
+        if key in ['build_image', 'subscriber', 'feature', 'cron', 'cron_schedule', 'cron_concurrency_policy', 'cron_starting_deadline_seconds']:
```

Comment:
we'd better make the line width be less than 89

Reply:
`cron_timezone`

Reply:
I consider making this list a standalone data structure.
We can place this list in prepare.py, beside [common_options](https://github.com/ofey404/tidb-testing/blob/fe049d8d780d2a87a1ca26a7b75111bf795f53af/tipocket-ctl/tpctl/prepare.py#L20).

Reply:
Well, it would cause a circular dependency